### PR TITLE
Schema: Use the latest cuetsy version

### PR DESCRIPTION
**Pull request by Clarity-89**
**What is this feature?**

Use the latest version of cuetsy with some fixes for generating nested TS types and defaulting to `unknown` vs `any`.

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer**:
With this update the default empty type used is `unknown` instead of `any`, which provides better type safety. After running the front end build there's no TS errors related to that, and it's easier to switch from using `unknown` to `any` than the other way around in the future.

### CODE REVIEW

File: packages/grafana-schema/src/raw/team/x/team_types.gen.ts [hunk 1]
```diff
@@ -9,16 +9,16 @@
 // Run 'make gen-cue' from repository root to regenerate.
 
 export enum Permission {
-  Admin = 4,
-  Editor = 2,
-  Viewer = 1,
+  admin = 4,
```
Comment (jackw) [inline packages/grafana-schema/src/raw/team/x/team_types.gen.ts:12]: How impactful are these changes for plugins? Should we worry about the levitate report? Or do plugins not use these yet? (They're public so I guess any plugin dev could pick them up if they wanted / needed to.)

Comment (Clarity-89) [reply packages/grafana-schema/src/raw/team/x/team_types.gen.ts]: I don't think this is used anywhere yet, but I've updated the type to be consistent with how a similar enum is defined on front end.

Comment (sdboyer) [reply packages/grafana-schema/src/raw/team/x/team_types.gen.ts]: > Should we worry about the levitate report?

This is interesting - i wonder if we could selectively configure levitate to automatically ignore things below a certain [maturity level](https://docs.google.com/document/d/1gPzSCQn-0Qg86PYpPcR-l_kTQ4Ba1wgnNLZFKPSe8yQ/edit#heading=h.480z6zsrjqh2)?

**Comment by sdboyer** [comment]
This LGTM, though as @joshhunt is my TS correctness spirit animal i would feel somewhat comfier if with an explicit +1 from him that he's happy with the `any` -> `unknown` switch, too

**Review by sdboyer — APPROVED** [review]

**Review by joanlopez — APPROVED** [review]
Looks good! I'll leave the `unknown` vs `any` debate for the experts, although what's mentioned in the pull request description makes as sense as my TS knowledge arrives to.

**Review by joshhunt — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**